### DISCUSSION TITLE
Support for table styles

### DISCUSSION THREAD

**Discussion by daun** [jacksleight/statamic-bard-texstyle#67] [General]
This looks fantastic! Before I dive too deep and splurge on the Pro version, does the addon currently support adding styles for tables? As in, checkboxes for 'Borderless' or 'Responsive' tables. Thanks :)

**Comment by jacksleight** [comment]
Yep you can use the [attributes](https://jacksleight.dev/docs/bard-texstyle/attributes) feature to do that. Define the options you want in the config and then use the attributes panel in the editor. Each custom attribute can be converted directly to an html tag attribute, class, or style (check the docs for full details).

Texstyle does not currently support custom table styles that can be toggled on/off through a button in the toolbar unfortunately (like the way custom heading styles work). This is due to a bug in Tiptap that prevents those styles from rendering.

**Reply by daun** [reply]
Thanks! Trying it out right now :)

**Reply by daun** [reply]
Works great, thanks!
